src/sentry/replays/consumers/recording_buffered.py

```diff
@@ -72,6 +77,13 @@
 from sentry.utils import json, metrics
 
 logger = logging.getLogger(__name__)
+logger.addFilter(
+    SamplingFilter(
+        {
+            "mobile_event": MOBILE_EVENT_SAMPLE_RATE,
```

Comment:
Why is this different from the other consumer?

Reply:
Are you suggesting we add rrweb_event_count here too? It's not currently logged in this file

Reply:
No just asking the question.  If we're not logging it we don't need to include it.

Reply:
We're only logging mobile_event in this one